src/hip/hip_build_utils.cpp

```diff
@@ -51,11 +51,14 @@ static const char* MIOPEN_CMP_FLAG_REL_PATHS[MIOPEN_CMP_MAX_Flag] = {
 // Global variable List to hold the Compiler Flag Paths.
 char MIOPEN_CMP_FLAGS[MIOPEN_CMP_MAX_Flag][PATH_MAX_LEN] = {0};
```

Comment:
I think we do not need this global anymore.

Reply:
Thanks for the comments, updated the changes in latest version